sonic-slave/Dockerfile

```diff
@@ -250,7 +250,7 @@ RUN apt-get update && apt-get install -y \
 RUN apt-get -y build-dep linux
 
 # For gobgp build
-RUN export VERSION=1.8.3 \
```

Comment:
Any reason not using 1.11.2?

Reply:
No specific concrete reason.   1.11.1 is recent enough. could change it to 1.11.2 which was release a few days ago.

Reply:
I saw you upgraded to 1.11.2, since 1.11.2 is mostly backport bug fixes comparing to 1.11.1, this should be a good move.